### ISSUE TITLE
Make GIF images more accessible for users

### ISSUE THREAD

**Issue by fabiankaegy** [WordPress/gutenberg#55538]
Currently, when you insert an animated GIF image into your site, it just cycles through its animation infinitely. This is the default browser behavior for GIFs however, it isn't great for users that have difficulty with movement and or blinking lights. 

Sadly browsers don't automatically respect `prefers-reduced-motion` to make GIFs pausable. 

Because of that it would be great for WordPress itself to implement a solution to make GIFs on the web more accessible to all by implementing the guidance outlined in WCAG

https://www.w3.org/TR/WCAG20-TECHS/G152.html

**Comment by fabiankaegy** [comment]
Tagging @00travelgirl00 since your Talk at WordCamp Germany 2023 inspired the creation of this issue

**Comment by joedolson** [comment]
I'm not sure this is an editor issue; the user doesn't need to make a decision about whether a GIF will be animated infinitely, this should be something that can be supported globally by WordPress. There's already code to handle this in the admin for plugin icons; although that may not be the same option we'd want on the front end. See https://core.trac.wordpress.org/changeset/56541
